meson.build

```diff
@@ -0,0 +1,74 @@
+## Experimetal meson build system for dinit
+project(
+    'dinit', 
+    'cpp', 
+    version : '0.16.0',
```

Comment:
Is there any way that you can avoid hardcoding `version` here?

Reply:
(it's better to use from `build/version.conf`. Having it here means this needs to be updated each release).